### DISCUSSION TITLE
I forgot my last two words for my pi wallet passphrase

### DISCUSSION THREAD

**Discussion by Thefallenangel666-byte** [community/community#134926] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I remember my 22 worlds how I can get last 2 words 
Please help me

**Comment by LiteBrite82** [comment]
Hi @azzujazzy ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
